src/hacbs/components/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Dropdown, DropdownItem, KebabToggle } from '@patternfly/react-core';
+import { RowFunctionArgs, TableData } from '../../../shared/components/table';
+import { Timestamp } from '../../../shared/components/timestamp/Timestamp';
+import { PipelineRunKind } from '../../types';
+import { calculateDuration } from '../../utils/pipeline-utils';
+import { pipelineRunTableColumnClasses } from './PipelineRunListHeader';
+
+const PipelineListRow: React.FC<RowFunctionArgs<PipelineRunKind>> = ({ obj }) => {
+  const [kebabOpen, setKebabOpen] = useState(false);
+
+  const capitalize = (label: string) => {
+    return label.charAt(0).toUpperCase() + label.slice(1);
+  };
+  return (
+    <>
+      <TableData className={pipelineRunTableColumnClasses.name}>
+        <Link to={`/app-studio/pipelineruns?name=${obj.metadata.name}`} title={obj.metadata.name}>
+          {obj.metadata.name}
+        </Link>
+      </TableData>
+      <TableData className={pipelineRunTableColumnClasses.started}>
+        <Timestamp
+          timestamp={typeof obj.status?.startTime === 'string' ? obj.status.startTime : ''}
+        />
+      </TableData>
+      <TableData className={pipelineRunTableColumnClasses.duration}>
+        {calculateDuration(
+          typeof obj.status.startTime === 'string' ? obj.status.startTime : '',
+          typeof obj.status.completionTime === 'string' ? obj.status.completionTime : '',
```

Comment:
`obj.status` is likely the cause of the error according to the stack trace because optional chaining isn't used consistently.

Reply:
added optional chaining to the rows

@MariaLeonova thanks, i fixed a couple of occurrences i found of 'PipelineRuns' and changed them to 'Pipelineruns' just so we're consistent. we can revisit if we need to. as for sorting, i haven't added filter/sorting components to the list/details yet, currently i just do a default sort of the list by creation timestamp date.